### ISSUE TITLE
Not able to set the legend position (MapChart & Angular)

### ISSUE THREAD

**Issue by sschrijver** [amcharts/amcharts4#608]
First of all: my compliments on the new AmCharts library, very cool new charts and features!

Currently, I am implementing AmCharts 4 on a new project we're working on.
At one page there is a 'MapChart' displayed, with a 'Legend'.

Default (without setting any position) the legend position is displaying is above the map (so, position: top).
Strange thing is, that it's not matching the documentation of the position property:
```/**
     * Position of the legend.
     *
     * Options: "left", "right", "top", "bottom" (default), or "absolute".
     *
     * @default "bottom"
     * @param {LegendPosition}  value  Position
     */
    position: LegendPosition;
```

We want to display the legend below or on the bottom of the MapChart.
According to the documentation that should be default and it is default on another chart displayed on the page (which is a XYChart).

Adding ```chart.legend.position = 'bottom'``` isn't changing anything either.
I'm not able to test it on the CodePens shown on the AmCharts docs either, because it seems that none of the MapChart CodePens are working.
**Edit:** Apparantly the CodePens do work when I open them in Incognito mode, but in my normal browser I get the following error on a CodePen containing a MapChart.

``` 
Uncaught TypeError: Object prototype may only be an Object or null: undefined
    at setPrototypeOf (<anonymous>)
    at Object.e.c (core.js:1)
    at MapImage.ts:72
    at Object.208 (MapImage.ts:72)
    at r (core.js:1)
    at Object.555 (MapChart.ts:1)
    at r (core.js:1)
    at Object.554 (maps.ts:1)
    at r (core.js:1)
    at Object.553 (maps.js:1)

Uncaught TypeError: Cannot read property 'call' of undefined
    at r (core.js:1)
    at Object.607 (animated.js:1)
    at r (core.js:1)
    at Object.606 (animated.js:1)
    at r (core.js:1)
    at Object.605 (animated.js:1)
    at r (core.js:1)
    at window.webpackJsonp (core.js:1)
    at animated.js:1

Uncaught ReferenceError: am4maps is not defined
    at pen.js:15
```

Anyone else facing this problem? Or anyone who know how to solve this?

**Comment by xorspark** [comment]
You'll need to clear your cache for the CodePen demos to work since our new releases on our CDN don't always get loaded correctly.

I can confirm that bottom and right positions don't work, but top and left do. The screenshot on the demo page shows that position right worked at one point, so it's likely a bug.

[Repro codepen](https://codepen.io/team/amcharts/pen/acdc5e561f19696cd845d4135fc68b1c/)

**Comment by martynasma** [comment]
Fixed in [4.0.3](https://github.com/amcharts/amcharts4/releases/tag/4.0.3).

## [4.0.3] - 2018-11-30

### Added
- Ability to set default locale via `am4core.options.defaultLocale`. When set, all new charts will automatically assume that locale, freeing you from setting locale for each chart individually.
- `reverseOrder` added to `Container`. Allows reversing order of its `children`.

### Changed
- Moved `Legend` of a `MapChart` from `chartAndLegendContainer` to `chartContainer`. In order to position legend on map chart, use `legend.align` / `legend.valign` properties.
- Default value of `showOnInit` on any `Series` is now `false`. Except if you are using "animated" theme, which sets it to `true` by default.

### Fixed
- JSON: Using `type: "Sprite"` in JSON config was not creating elements properly.
- Initial chart rendering optimized Charts should build faster.
- Some more performance optimizations.
- Setting `hiddenInLegend` on `Series` after chart was already built had no effect.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.

**Comment by martynasma** [comment]
Use like this:

```
chart.legend.align = "right";
chart.legend.valign = "bottom";
```
